### ISSUE TITLE
Can yacy look for updates at https://release.yacy.net/?

### ISSUE THREAD

**Issue by kotenok2000** [yacy/yacy_search_server#543]

**Comment by Orbiter** [comment]
Yes, I made a first step in https://github.com/yacy/yacy_search_server/commit/b1199e97f82376458e3693097a85a779ffe62175

However, this can not be tested without tension in development environments. By the nature of this feature it is only possible to test this in a future version where the version of this commit is able to load from a future version.
